tendrl/monitoring_integration/grafana/alert_organization.py

```diff
@@ -64,15 +64,15 @@ def create_organization():
     resp = json.loads(resp)
     if "id" in resp:
         org_id = resp['id']
-        msg = ("alert organization with name %s " +
+        msg = ("alert organization with name %s "
```

Comment:
as per pep8 check, we don't need to give + symbol in between brackets for string concatenation